lib/auth/apiserver.go

```diff
@@ -89,9 +88,6 @@ func NewAPIServer(config *APIConfig) (http.Handler, error) {
 	srv.POST("/:version/kube/csr", srv.withAuth(srv.processKubeCSR))
 
 	// Operations on certificate authorities
-	srv.GET("/:version/domain", srv.withAuth(srv.getDomainName))    // DELETE IN 11.0.0 REST method replaced by gRPC
```

Comment:
Should we document somewhere that we're removing some endpoints?

Reply:
No, we don't typically because we don't document a public web API.